tests/test_load_waypoint_name_to_coordinates_map.py

```diff
@@ -4,6 +4,7 @@
 import pathlib
 
 from modules import load_waypoint_name_to_coordinates_map
+from modules.waypoint import Waypoint
```

Comment:
Import namespace over class, but keep as absolute import.

The reason for keeping as an absolute import is because this file will be run as a test script (rather than a library that's being imported to somewhere).